utils/logger/zap_logger.go

```diff
@@ -60,6 +60,11 @@ func (z *ZapLogger) WithField(key string, value interface{}) Logger {
 	return &ZapLogger{l: z.l.With(key, value)}
 }
 
+// SetLevel sets log level.
+func (z *ZapLogger) SetLevel(level Level) {
+	// TODO: implement it.
```

Comment:
Are we gonna ever implement this, or is it just stub so we comply with Logger interface?

Reply:
We will implement it once we start using dbaas-controller in platform.
For now it's just to comply with Logger interface

Reply:
ok